books/RayTracingInOneWeekend.html

```diff
@@ -2811,25 +2811,24 @@
 You now have a cool ray tracer! What next?
 
   1. Lights. You can do this explicitly, by sending shadow rays to lights. Or it can be done
```

Comment:
Comma before conjunction: “... sending shadow ray to lights, or it can be done ...”.

Reply:
Suggesting that we use a semicolon instead of a comma to join these sentences, in order to more clearly separate the "explicit" and "implicit" approaches to lights.